src/Task.cc

```diff
@@ -112,13 +112,15 @@ void Task::detach() {
 }
 
 void Task::reenable_cpuid_tsc() {
+#if defined(__i386__) || defined(__x86_64__)
```

Comment:
Instead of conditional compilation how about just checking `if (Arch::is_x86ish())`?

Reply:
Fewer #ifdefs means less cross-platform compilation bustage.

Reply:
Done.